### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by MariaShebeko**

### CODE REVIEW

File: src/sass/layout/_header.scss [hunk 1]
```diff
@@ -84,45 +83,14 @@
   padding-bottom: 90px;
 }
 
+.header__title {
```
Comment (Fosa1990) [inline src/sass/layout/_header.scss]: position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  border: 0;
  padding: 0;

  white-space: nowrap;
  clip-path: inset(100%);
  clip: rect(0 0 0 0);
  overflow: hidden;

File: src/sass/layout/_header.scss [hunk 2]
```diff
@@ -84,45 +83,14 @@
...
+  display: none;
```
Comment (Fosa1990) [inline src/sass/layout/_header.scss]: delete ===>  display: none;

**Review by Fosa1990 — CHANGES_REQUESTED** [review]
fix pls display: none; to .visually-hidden

File: src/sass/layout/_header.scss (hunk 1 above)
Comment (MariaShebeko) [reply src/sass/layout/_header.scss]: +

File: src/sass/layout/_header.scss (hunk 2 above)
Comment (MariaShebeko) [reply src/sass/layout/_header.scss]: +

**Review by Fosa1990 — APPROVED** [review]

**Comment by Fosa1990** [comment]
ok

**Pull request merged**